### DISCUSSION TITLE
Can I save the spin-up results of cwatm-modflow?

### DISCUSSION THREAD

**Discussion by hsk0059** [iiasa/CWatM#154] [Q&A]
It will take a longer time to run cwatm-modflow when compared to the standard cwatm. I wanna save the result of the end of the spinning-up period like the stanard cwatm program, and use it to perform my next cwatm-modflow running. However, I donot find any groundwater information (or variables) in the save .nc file. Is it possible for me to save the spin-up results of cwatm-modflow?

**Comment by mikhailsmilovic** [comment]
Yes, we can save the water table from the end of a run and set it as our initial water table.

Include the output variable **modflow_watertable** in the list for  _OUT_MAP_TotalEnd_
`OUT_MAP_TotalEnd = modflow_watertable`

Then state to be loading an initial water table, and point to this file in the settings at **init_water_table**: 

```
load_init_water_table = True
init_water_table = PATHTOFOLDER/modflow_watertable_totalend.nc
```

Saving and loading an initialization file for all the other CWatM variables is the same as standard CWatM. For completeness,

```
save_inital = True
StepInit = DATE_WITHIN_SIMULATION_PERIOD
```
Then, after the init file is created
```
load_inital = True
initLoad = PATH_TO_INIT_FILE
# Then, to avoid overwriting your init file, unless intended to, or the dates in StepInit are different
save_inital = False
```
